src/gui/dialogs/editor/edit_unit.cpp

```diff
@@ -109,11 +108,11 @@ void editor_edit_unit::pre_show(window& win) {
 
 	button& load = find_widget<button>(&win, "load_unit_type", false);
 	std::stringstream tooltip;
-	tooltip << vgettext_impl("wesnoth", "Hotkey(s): ",  {{}});
+	tooltip << t_string("Hotkey(s): ", "wesnoth");
```

Comment:
This (and the existing implementation) only get into the .po files because the same string is in theme.cpp, but it's not a regression so let's not worry about it for now.